app/api/v1/endpoints/auth.py

```diff
@@ -0,0 +1,57 @@
+from fastapi import APIRouter, Depends, Response, Request, Form
+from sqlalchemy.orm import Session
+from app.schemas.user import (
+    UserCreateUpdate,
+    UserInDB,
+    UserInResponse,
+    UserLogin,
+    TokenResponse,
+    TokenRequest,
+)
+from app.crud.user import create_user, login_user
+from app.api import deps
+from app.limiter import limiter
+from app.core.config import settings
+
+router = APIRouter()
+
+
+@router.post("/register", response_model=UserInDB)
+@limiter.limit(f"{settings.MAX_AUTH_REQUESTS_COUNT}/minute")
+def register_user_endpoint(
+    request: Request, user_in: UserCreateUpdate, db: Session = Depends(deps.get_db)
+):
+    """
+    Register a new user.
+    """
+    return create_user(db, user_in)
```

Comment:
Create and share standard Response structure in chat.

Reply:
Currently we are using default Response for this app which doesn't require custom response